scarlet/display.py

```diff
@@ -338,25 +352,34 @@ def show_scene(
         if np.all(mask == 0):
             mask = None
 
-    model_frame = sources[0].frame
-    model = np.zeros(model_frame.shape)
-    for src in sources:
-        model += src.get_model(frame=model_frame)
+    if isinstance(sources, LiteBlend):
```

Comment:
I'm not a fan of having `lite` classes in main branch code. Can this be moved to `lite/display.py`?

Reply:
Absolutely. This was done originally to get this working for Erin, but I agree that it really belongs in the lite module.